fourmolu.yaml

```diff
@@ -0,0 +1,14 @@
+indentation: 2
+function-arrows: trailing
+comma-style: leading
+import-export-style: diff-friendly
+indent-wheres: true
+record-brace-space: true
+newlines-between-decls: 1
+haddock-style: single-line
+haddock-style-module:
+let-style: auto
+in-style: right-align
+unicode: never
```

Comment:
if I've understood correctly from the docs, this means to "**never** complain about unicode".

Reply:
Nah, as far as I understand it, this means to not do the autoconversion to unicode symbols: `->` -> `→`, `::` -> `∷`, etc. 
https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/unicode_syntax.html

In other words when this setting is set to `auto`, then it will detect if `UnicodeSyntax` extension is enabled and will automatically convert to unicode. `never` means do nothing.